### PR TITLE
updated choose command

### PR DESCRIPTION
**Pull request by Progame03**
I've updated the choose command to take up to ten possible choices (similar to Dank Memer)

### CODE REVIEW

File: src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java [hunk 1]
```diff
@@ -4,11 +4,14 @@
 import at.xirado.bean.command.SlashCommandContext;
 import net.dv8tion.jda.api.entities.Member;
 import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
 import net.dv8tion.jda.api.interactions.commands.OptionType;
 import net.dv8tion.jda.api.interactions.commands.build.CommandData;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import javax.swing.text.html.Option;
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java:13]: Please remove

File: src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java [hunk 2]
```diff
@@ -18,6 +21,14 @@ public ChooseCommand()
         setCommandData(new CommandData("choose", "Chooses between 2 things.")
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java:18]: The description should be updated

File: src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java [hunk 3]
```diff
@@ -26,15 +37,20 @@ public ChooseCommand()
...
-        String secondOption = event.getOption("2nd").getAsString();
-        int i = ThreadLocalRandom.current().nextInt(2);
-        if (i == 0)
-        {
-            ctx.reply("I choose... " + firstOption).queue();
-        } else
-        {
-            ctx.reply("I choose... " + secondOption).queue();
+        List<OptionMapping> chooseOptions = event.getOptions();
+        if(checkOptions(chooseOptions)){
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java:41]: This isn't necessary. The `SlashCommandContext` methods are configured to not allow those mentions

**Comment by Progame03** [comment]
On it, doing the changed asap

**Comment by Progame03** [comment]
done

File: src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java [hunk 4]
```diff
@@ -4,20 +4,30 @@
...
 
 public class ChooseCommand extends SlashCommand
 {
     public ChooseCommand()
     {
-        setCommandData(new CommandData("choose", "Chooses between 2 things.")
+        setCommandData(new CommandData("choose", "Chooses between up to 10 things.")
                 .addOption(OptionType.STRING, "1st", "First argument.", true)
                 .addOption(OptionType.STRING, "2nd", "Second argument.", true)
+                .addOption(OptionType.STRING, "3rd", "Third argument.", false)
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/ChooseCommand.java:23]:
```suggestion
                .addOption(OptionType.STRING, "3rd", "Third argument.")
```

This is idempotent

**Review by Xirado — CHANGES_REQUESTED** [review]

**Comment by Progame03** [comment]
changes done

**Review by Xirado — APPROVED** [review]

**Pull request merged**